### PR TITLE
wordpress service + event-emitters

### LINKED ISSUE #454: Replace setInterval
using `setInterval` can be a huge issue described in multiple articles.
just search "nodejs why setInterval is bad".
a common solution is something like https://www.npmjs.com/package/node-cron, or everything that is based on https://nodejs.org/api/events.html#class-eventemitter

### PR DESCRIPTION
**Pull request by fcaps**
closes #454
related to #480

### CODE REVIEW

File: lib/WordpressRepository.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
+const {convert} = require("url-slug");
+
+class WordpressRepository {
+    constructor(wordpressClient) {
+        this.wordpressClient = wordpressClient
+    }
+    async fetchNews() {
+        let response = await this.wordpressClient.get('/wp-json/wp/v2/posts/?per_page=100&_embed&_fields=_links.author,_links.wp:featuredmedia,_embedded,title,content.rendered,date,categories&categories=587')
```
Comment (Brutus5000) [inline lib/WordpressRepository.js:8]: I think the categories url was configurable via env?
But I see so many categories now, I'm no longer sure.
Maybe we should at least put the ids as const variables at the top of the file for better naming in the actual urls?

Comment (fcaps) [reply lib/WordpressRepository.js:8]: was just copy&pasta, https://github.com/FAForever/website/pull/487/files#diff-7e4783341040a4ba6619cd0157ccb8d361c450d4ac84148a2fe589414db924ddL22

maybe we can optimize it later? there are for sure other problems, like "options/fields" not used...

File: routes/views/account/get/report.js [hunk 2]
```diff
@@ -101,23 +101,7 @@ exports = module.exports = function (req, res) {
       }
 
       locals.reportable_members = {};
-      const recentMembersPath = 'members/recent.json';
-      if (fs.existsSync(recentMembersPath)){
```
Comment (fcaps) [inline routes/views/account/get/report.js:105]: we got this covered in #233, when the m2m is ready

File: scripts/cron-jobs.js [hunk 3]
```diff
@@ -1,16 +1,27 @@
...
+    wordpressService.getNewshub(true).then(() => successHandler('getNewshub')).catch((e) => errorHandler(e, 'getNewshub'))
+    wordpressService.getContentCreators(true).then(() => successHandler('getContentCreators')).catch((e) => errorHandler(e, 'getContentCreators'))
+    wordpressService.getTournamentNews(true).then(() => successHandler('getTournamentNews')).catch((e) => errorHandler(e, 'getTournamentNews'))
+    wordpressService.getFafTeams(true).then(() => successHandler('getFafTeams')).catch((e) => errorHandler(e, 'getFafTeams'))
+}
+
+module.exports = async () => {
+    await warmupWordpressCache()
+    
+    const wordpressScheduler = new Scheduler('createWordpressCaches', warmupWordpressCache, 60 * 59 * 1000)
```
Comment (fcaps) [inline scripts/cron-jobs.js:25]: i'll make all the "magic" numbers into a config once it is somewhat stable and does not change so often.
don't wanna add env-vars directly

File: templates/views/newshub.pug [hunk 4]
```diff
@@ -19,8 +19,6 @@ html(lang='en')
 
     //- Customise the stylesheet for your site by editing /public/styles/site.sass
     link(href="/styles/css/site.min.css?version="+Date.now(), rel="stylesheet")
-    script(async, defer, data-domain="faforever.com", src="https://plausible.faforever.com/js/plausible.js")
-    script(src="https://kit.fontawesome.com/33e7e258d3.js" crossorigin='anonymous')
```
Comment (fcaps) [inline templates/views/newshub.pug:23]: plausible http 404
kit: http 403

don't know why this was loaded, but the newshub still looks okay

Comment (Brutus5000) [reply templates/views/newshub.pug:23]: Plausible is currently disabled on the server.

**Review by Brutus5000 — APPROVED** [review]

**Pull request merged**